manager/controllers/utils/utils.go

```diff
@@ -11,6 +11,13 @@ import (
 	"runtime"
 )
 
+const (
+	StepNameHashLength       = 10
```

Comment:
all constants for the package internal usage -> low case

Reply:
Missed that, I'll fix it